Samples/Islands/Lottie/LottieIsland/LottieContentIsland.idl

```diff
@@ -0,0 +1,40 @@
+﻿namespace LottieIsland
+{
+    [default_interface]
+    runtimeclass LottieContentIsland
+    {
+        LottieContentIsland(Microsoft.UI.Composition.Compositor compositor);
+
+        Microsoft.UI.Content.ContentIsland Island{ get; };
+
+        Microsoft.UI.Xaml.Controls.IAnimatedVisualSource AnimatedVisualSource;
```

Comment:
Hmmm. We'll need to follow-up on this. We don't want to bring in Xaml types, as it will cause conflicts and upside-down dependencies.

Reply:
Shoot, I was worried about that. I thought since it's just an interface, it might be alright. It doesn't actually load MUX.dll

Reply:
@jeffstall @iablaauw-MS Do we think we should block on this? I think that fixing this would require pushing changes into the Lottie-Windows repo.